test/Index/roles.swift

```diff
@@ -499,3 +499,18 @@ protocol WithPrimary<Assoc> {
 // CHECK: [[@LINE-1]]:18 | type-alias/associated-type/Swift | Assoc | {{.*}} | Def,RelChild | rel: 1
 // CHECK-NEXT: RelChild | protocol/Swift | WithPrimary | {{.*}}
 }
+
+struct Root {} // CHECK: [[@LINE]]:8 | struct/Swift | Root | [[Root_USR:.*]] | Def | rel: 0
+
+typealias Alias = Root // CHECK: [[@LINE]]:11 | type-alias/Swift | Alias | [[Alias_USR:.*]] | Def | rel: 0
+
+extension Alias {
+  // CHECK: [[@LINE-1]]:11 | type-alias/Swift | Alias | [[Alias_USR:.*]] | Ref | rel: 0
+  // CHECK: [[@LINE-2]]:11 | struct/Swift | Root | [[Root_USR:.*]] | Ref,Impl,RelExt | rel: 1
+  func empty() {}
+}
+
+extension Root {
+  // CHECK: [[@LINE-1]]:11 | struct/Swift | Root | [[Root_USR:.*]] | Ref,RelExt | rel: 1
```

Comment:
I'd hope we have a test for this already, suppose there's no harm in an extra one though 😅

Reply:
hrm i'll drop it actually though, I think there was a point in having it in an earlier change